css/style.css

```diff
@@ -59,18 +67,140 @@ body {
 
 /* Center the container */
 #container {
-  width: 960px;
   margin: auto;
+  padding-right: 20px;
+  padding-left: 40px;
+
 }
 
-header {
+div{
+
+  background-color: #033048;
+  
+  }
+.leftfloat{
+   width:700px;
+   color:white;
+   font-size: 40px;
+   font-weight: bold;
 
 }
+.leftfloat2{
+   width:700px;
+   color:#606161;
+   font-size: 12px;
```

Comment:
Because the `font-size` is already set to `12px` on the body, you shouldn't ned to respecify this.